### PR TITLE
Made ip_no mandatory in ConsultationForm

### LINKED ISSUE #2384: IP Number : Mandatory field
IP Number field at https://care.coronasafe.in/facility/7a560e62-6aae-4293-8f1a-0ba840e3449f/patient/192fdee9-397f-49bd-833a-10f700416da8/consultation/d794ba54-da42-4238-8ba9-0b628ca60069/update

is to be made mandatory

### PR DESCRIPTION
**Pull request by khavinshankar**
Fixes #2384 

image

### CODE REVIEW

File: src/Components/Facility/ConsultationForm.tsx [hunk 1]
```diff
@@ -820,7 +827,7 @@ export const ConsultationForm = (props: any) => {
               </div>
 
               <div id="ip_no-div">
-                <InputLabel id="refered-label">IP number</InputLabel>
+                <InputLabel id="refered-label">IP number*</InputLabel>
```
Comment (developedBySJ) [inline src/Components/Facility/ConsultationForm.tsx:830]: @khavinshankar You can use the `required` prop
https://mui.com/material-ui/api/input-label/

Comment (khavinshankar) [reply src/Components/Facility/ConsultationForm.tsx:830]: @developedBySJ  I did think of that, but do `required` attribute help with user friendly message?

**Comment by nihal467** [comment]
@khavinshankar @developedBySJ @gigincg is it good for testing ?

**Comment by khavinshankar** [comment]
@nihal467 yes it is ready for testing now

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/ConsultationForm.tsx (hunk 1 above)
Comment (developedBySJ) [reply src/Components/Facility/ConsultationForm.tsx:830]: @khavinshankar What do you mean by user-friendly message?

**Pull request merged**